github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -257,7 +257,6 @@ presubmits:
       preset-shared-images: "true"
     max_concurrency: 2
```

Comment:
If this is to run and be required on every kubevirt PR - we may need to increase the max concurrency to 4 or 6